Package.swift

```diff
@@ -4,30 +4,23 @@ import PackageDescription
 
 let package = Package(
     name: "mParticle-Kochava",
-    platforms: [
-            .iOS("10.3"),
-            .tvOS("10.2"),
-        ],
+    platforms: [ .iOS("10.3") ],
     products: [
         .library(
             name: "mParticle-Kochava",
             targets: ["mParticle-Kochava"]),
     ],
     dependencies: [
-        .package(name: "mParticle-Apple-SDK", url: "git@github.com:mParticle/mparticle-apple-sdk.git", from: "8.2"),
-        .package(
-                    name: "KochavaCore",
-                    url: "https://github.com/Kochava/Apple-SwiftPackage-KochavaCore",
-                    from: "4.0.0"
-                ),
-        .package(name: "KochavaTracker", url: "https://github.com/Kochava/Apple-SwiftPackage-KochavaTracker.git", from: "4.0.0"),
+        .package(name: "mParticle-Apple-SDK",
+                 url: "https://github.com/mParticle/mparticle-apple-sdk",
```

Comment:
any reason for the change here to allow less than 8.2? if anything should we update it to be 8.3?

Reply:
Leftover from me testing out fixes. I agree with setting it to 8.3.0